PKGBUILD

```diff
@@ -125,6 +129,22 @@ _package() {
 
   # add System.map
   install -D -m644 System.map "${pkgdir}/boot/System.map-${_kernver}"
+
+  # add /etc/initcpio/install/bcachefs
+  install -D -m644 bcachefs-install "${pkgdir}/etc/initcpio/install/bcachefs"
+
+  # add /etc/initcpio/hooks/bcachefs
+  install -D -m644 bcachefs-hooks "${pkgdir}/etc/initcpio/hooks/bcachefs"
+
+  # edit /etc/mkinitcpio.conf to include bcachefs hooks if it doesn't already
+  # and create a backup of original at /etc/mkinitcpio.conf.bak
+  if ! grep -rnw '/etc/mkinitcpio.conf' -e 'bcachefs' >/dev/null 2>&1; then
+    echo "creating /etc/mkinitcpio.conf.bak and adding bcachefs hook to /etc/mkinitcpio.conf"
+    sed -i.bak 's/keyboard/bcachefs &/g' /etc/mkinitcpio.conf
+  else
+    echo "/etc/mkinitcpio.conf already contains bcachefs hook"
+  fi
```

Comment:
this will apply changes to the system the build is running on, you can't just put this into the `PKGBUILD`

Reply:
I can remove that. I don't think the hook is necessary if it's an unencrypted volume. Since encryption isn't default, the user will be manually setting it up anyway and adding a hook to `/etc/mkinitcpio.conf` isn't that much of a bother.